content/blog/2021/06/2020-06-03-digester-removal.adoc

```diff
@@ -0,0 +1,37 @@
+---
+layout: post
+title: "commons-digester library removal (breaking changes!)"
+tags:
+- jenkins
+- core
+- developer
+author: batmat
+description: "The commons-digester:2.1 library got removed from Jenkins Core. A few plugins will be broken with the upcoming release."
+---
+
+Keeping up with our goal to clean up some of the technical debt inside Jenkins Core, we are happy to report we were able to remove a long-deprecated library.
+
+link:https://github.com/jenkinsci/jenkins/pull/5320[Jenkins does not depend anymore] on link:https://search.maven.org/artifact/commons-digester/commons-digester/2.1/jar[commons-digester:2.1], a version that got released in 2010.
+
+Here is the list of plugins that did _not_ get fixed at the time of writing:
```

Comment:
these have fixes but just not publicly available right?
If so should we be clearer on this, something like

```suggestion
Here is the list of plugins that did _not_ have their fix released by a maintainer at the time of writing:
```